package.json

```diff
@@ -56,7 +56,7 @@
     "passport": "^0.4.0",
     "passport-oauth2": "^1.4.0",
     "prop-types": "^15.7.2",
-    "query-string": "^6.4.0",
+    "query-string": "^5.1.1",
```

Comment:
downgrade according to release notes at https://github.com/sindresorhus/query-string/releases/tag/v6.0.0